src/signals/settings/users/containers/Overview/__tests__/Overview.test.js

```diff
@@ -11,15 +11,24 @@ import {
 import { history as memoryHistory, withCustomAppContext } from 'test/utils';
 
 import usersJSON from 'utils/__tests__/fixtures/users.json';
+import inputRolesSelectorJSON from 'utils/__tests__/fixtures/inputRolesSelector.json';
 import { USER_URL } from 'signals/settings/routes';
 import configuration from 'shared/services/configuration/configuration';
 import * as constants from 'containers/App/constants';
-import * as settingsActions from 'signals/settings/actions';
 import * as reactRouter from 'react-router-dom';
 import * as appSelectors from 'containers/App/selectors';
-import UsersOverview from '..';
+import { setUserFilters } from 'signals/settings/actions';
+import SettingsContext from 'signals/settings/context';
+import * as rolesSelectors from 'models/roles/selectors';
+
+import usersFilterInactiveJSON from 'utils/__tests__/fixtures/usersFilterInactive.json';
+import usersFiltersActiveJSON from 'utils/__tests__/fixtures/usersFiltersActive.json';
+import usersFiltersActiveAndRoleJSON from 'utils/__tests__/fixtures/usersFiltersActiveAndRole.json';
+import usersFiltersDisabledJSON from 'utils/__tests__/fixtures/usersFiltersDisabled.json';
+import usersFiltersInactiveAndRoleJSON from
```

Comment:
Can't you use one fixture and perform filters on that set in this test? Then we don't need as many fixtures.
🤷‍♂

Reply:
Will do.